ui/components/multichain/nft-item/nft-item.js

```diff
@@ -16,27 +16,34 @@ import {
   JustifyContent,
 } from '../../../helpers/constants/design-system';
 import {
+  getCurrentNetwork,
   getIpfsGateway,
   getOpenSeaEnabled,
   getTestNetworkBackgroundColor,
 } from '../../../selectors';
+import useGetAssetImageUrl from '../../../hooks/useGetAssetImageUrl';
+import { getNftImageAlt } from '../../../helpers/utils/nfts';
 
-export const NftItem = ({
-  alt,
-  name,
-  src,
-  networkName,
-  networkSrc,
-  tokenId,
-  onClick,
-  clickable,
-  isIpfsURL,
-  badgeWrapperClassname,
-}) => {
+export const NftItem = ({ nft, onClick, clickable, badgeWrapperClassname }) => {
   const testNetworkBackgroundColor = useSelector(getTestNetworkBackgroundColor);
   const isIpfsEnabled = useSelector(getIpfsGateway);
   const openSeaEnabled = useSelector(getOpenSeaEnabled);
 
+  const { image, imageOriginal, name, tokenId } = nft;
+
+  const ipfsGateway = useSelector(getIpfsGateway);
+  const currentChain = useSelector(getCurrentNetwork);
```

Comment:
This line is problematic; we need a selector that mimics the return value of `getCurrentNetwork` but allows us to pass a chainId instead of assuming the current.